### PR TITLE
replace innards of vt_add_file_to_config with tidyselect functions

### PR DESCRIPTION
**Pull request by thebioengineer**
Ellis' suggestions around tidyselect usage

### CODE REVIEW

**Comment by thebioengineer** [comment]
@elimillera - what do you think of the suggestions?

**Review by mariev — APPROVED** [review]
added a test case but this looks good otherwise if we can get the conflicts resolved.

File: R/case.R [hunk 1]
```diff
@@ -64,7 +64,7 @@ vt_use_test_case <- function(name, username = vt_username(), title = NULL, open
   }
 
   # Add file to validation configuration
-  vt_add_file_to_config(filename = name, after = add_after, before = add_before)
```
Comment (elimillera) [inline R/case.R:67]: Kudos to you. I was playing around with the curly-curly but it was giving me trouble and just relied on lazy evaluation instead.

**Review by elimillera — APPROVED** [review]
I like it! Thanks @thebioengineer

**Pull request merged**